### PR TITLE
Them hien thi trang thai ung tuyen vao cac trang list job va detail job

### PR DESCRIPTION
**Pull request by nmt0607**
## Related Tickets:
- [#Ticket 32982](https://edu-redmine.sun-asterisk.vn/issues/32982)

## Notes Impacted Areas:
- N/A.

## Attachments:
- N/A.

### CODE REVIEW

**Comment by nmt0607** [comment]
ready

**Comment by nmt0607** [comment]
ready

File: app/Http/Controllers/JobController.php [hunk 1]
```diff
@@ -23,10 +23,12 @@ public function index()
         $skills = Tag::where('type', config('tag_config.skill'))->get();
         $langs = Tag::where('type', config('tag_config.language'))->get();
         $workingTimes = Tag::where('type', config('tag_config.working_time'))->get();
+        $appliedJobs = Auth::user()->jobs()->where('applications.status', 0)->get();
```
Comment (congnx-2423) [inline app/Http/Controllers/JobController.php]: cho 0 vào config hoặc enum, tránh hardcode

File: app/Http/Controllers/JobController.php [hunk 2]
```diff
@@ -69,9 +71,11 @@ public function show($id)
         }
         $jobCurrent = Job::where('id', $id)->get();
         $similarJobs = $similarJobs->diff($jobCurrent);
+        $appliedJobs = Auth::user()->jobs()->where('applications.status', 0)->get();
```
Comment (congnx-2423) [inline app/Http/Controllers/JobController.php]: tương tự

File: resources/views/listjob.blade.php [hunk 3]
```diff
@@ -65,13 +65,28 @@
...
+                                                        <ul class="dropdown-menu">
+                                                            <li>
+                                                                <a href="{{ route('cancel_apply', ['id' => $job->id]) }}">
+                                                                    <font class="apply-status">@lang('job.cancel_apply')</font>
+                                                                </a>
+                                                            </li>
+                                                        </ul>
+                                                    </li>
+                                                @else
+                                                    <form action="{{ route('apply', ['id' => $job->id]) }}" method="POST">
```
Comment (congnx-2423) [inline resources/views/listjob.blade.php:85]: method đâu em

**Comment by nmt0607** [comment]
ready

**Pull request merged**